### PR TITLE
Feature/coupon to db

### PR DESCRIPTION
**Pull request by tomdantuit**
Closes 11

### CODE REVIEW

File: src/main/java/com/e451/domain/Product.java [hunk 1]
```diff
@@ -3,46 +3,46 @@
...
     private String department;
     private String brand;
-    private String curr_size_of_product;
+    private String currentSizeOfProduct;
 
-    public Product(Long product_id, String commodity_desc, String sub_commodity_desc, String manufacturer, String department, String brand, String curr_size_of_product) {
-        this.product_id = product_id;
-        this.commodity_desc = commodity_desc;
-        this.sub_commodity_desc = sub_commodity_desc;
+    public Product(Long productId, String commodityDesc, String subCommodityDesc, String manufacturer, String department, String brand, String currentSizeOfProduct) {
```
Comment (johnmuller-8451) [inline src/main/java/com/e451/domain/Product.java]: This is an awfully long line of code. Why not
```
public Product(Long productId, String commodityDesc,
               String subCommodityDesc, String manufacturer, 
               String department, String brand,
               String currentSizeOfProduct) {
...
}
```

File: src/main/java/com/e451/model/ProductRecord.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    private String subCommodityDesc;
+    private String manufacturer;
+    private String department;
+    private String brand;
+    @Column(name="curr_size_of_product")
+    private String currentSizeOfProduct;
+
+    public ProductRecord() {}
+
+    public ProductRecord(Long productId, String commodityDesc, String subCommodityDesc, String manufacturer, String department, String brand, String currentSizeOfProduct) {
```
Comment (johnmuller-8451) [inline src/main/java/com/e451/model/ProductRecord.java]: This is a similarly long line of code to the one in src/main/java/com/e451/domain/Product.java. Maybe split it across a few lines.

File: src/test/java/com/e451/service/CampaignDescriptionServiceTest.java [hunk 3]
```diff
@@ -106,10 +106,10 @@ public void testCampaignDescriptionServiceCampaignDescriptionRecordCampaignDescr
                 , CampaignDescription.getDescription());
         Assert.assertEquals("The Start_Day was incorrectly converted"
                 , CAMPAIGN_DESCRIPTION_START_DAY1
-                , CampaignDescription.getStart_day());
+                , CampaignDescription.getStartDay());
```
Comment (johnmuller-8451) [inline src/test/java/com/e451/service/CampaignDescriptionServiceTest.java:109]: I know it makes things easier to compile when you comment things out, but I really, _really_ hate commas at the beginning of a line of code.

**Review by johnmuller-8451 — COMMENTED** [review]
Mostly just me being picky. We might have a chat about coding standards to settle on one way or another of breaking up lines, etc.

Not completely necessary to make these changes, but the code could be prettier if I had my way.

**Review by HLaiveling — APPROVED** [review]
Looks good.  Like the tests.

**Review by johnmuller-8451 — APPROVED** [review]

**Pull request merged**